terraform/aws/analytical-platform-data-engineering-sandbox-a/airflow-hmcts-sdp-load/iam-policies.tf

```diff
@@ -1,13 +1,130 @@
 data "aws_iam_policy_document" "airflow_hmcts_sdp_load" {
   statement {
-    sid    = "BucketAccess"
+    sid = "AllowListAllMyBuckets"
+    effect = "Allow"
+    actions = [
+      "s3:GetBucketLocation",
+      "s3:ListAllMyBuckets"
+    ]
+    resources = [
+      "*"
+    ]
+  }
+  statement {
+    sid = "AllowGetPutObject"
+    effect = "Allow"
+    actions = [
+      "s3:GetObject",
+      "s3:PutObject"
+    ]
+    resources = [
+      "arn:aws:s3:::aws-athena-query-results-*"
```

Comment:
Are you able to constrain this further?

Reply:
updated to point to specific bucket-name